src/Playwright.Tests/PageEvaluateTests.cs

```diff
@@ -547,24 +547,62 @@ public async Task ShouldNotLeakHandles()
     }
 
     [PlaywrightTest("page-evaluate.spec.ts", "should evaluate exception with a function on the stack")]
-    [Ignore("todo https://github.com/microsoft/playwright-dotnet/issues/2947")]
     public async Task ShouldEvaluateExceptionWithAFunctionOnTheStack()
     {
         var exception = await Page.EvaluateAsync<PlaywrightException>(@"() => {
             return (function functionOnStack() {
                 return new Error('error message');
             })();
         }");
-        Assert.Equals("error message", exception.Message);
-        StringAssert.Contains("functionOnStack", exception.StackTrace);
+        StringAssert.Contains("Error: error message", exception.Message);
+        StringAssert.Contains("functionOnStack", exception.Message);
```

Comment:
Shouldn't it be in exception.StackTrace ?

Reply:
We don't re-serialise the JS error into a .NET StackTrace property yet, I can try doing it tho, we only do message as of today.